alerter/BinancePumpAndDumpAlerter.py

```diff
@@ -70,30 +70,28 @@ def extract_ticker_data(symbol, assets):
 
     @staticmethod
     def create_new_asset(symbol, chart_intervals):
-        asset = {"symbol": symbol, "price": []}
+        asset = {"symbol": symbol, "price": [], "volume": []}
 
         for interval in chart_intervals:
             asset[interval] = {}
             asset[interval]["change_current"] = 0
             asset[interval]["change_last"] = 0
+            asset[interval]["change_volume"] = 0
 
         return asset
 
-    def retrieve_exchange_assets(self, api_url, retry_interval):
-        while True:
-            try:
-                self.logger.debug(
-                    "Retrieving price information from the ticker. ApiUrl: %s.", api_url
-                )
-                return requests.get(api_url).json()
-            except Exception as e:
-                self.logger.error(
-                    "Issue occurred while getting prices. Retrying in %ss. Error: %s.",
-                    retry_interval,
-                    e,
-                    exc_info=True,
-                )
-                sleep(retry_interval)

```

Comment:
With the removal of the retry_interval and loop, if we fail to get a response from the rest api will the program crash?